src/main/scala/viper/server/protocol/ViperIDEProtocol.scala

```diff
@@ -121,15 +121,17 @@ object ViperIDEProtocol extends akka.http.scaladsl.marshallers.sprayjson.SprayJs
     override def write(obj: EntitySuccessMessage): JsObject = {
       JsObject(
         "entity" -> obj.concerning.toJson,
-        "time" -> obj.verificationTime.toJson)
+        "time" -> obj.verificationTime.toJson,
+        "cached" -> obj.cached.toJson)
```

Comment:
Please file an issue that describes how the VS Code plugin should adapt to these changes.